vivisect/qt/funcgraph.py

```diff
@@ -167,6 +168,12 @@ def __init__(self, vw, vwqgui):
         self.vwqgui = vwqgui
         self._last_viewpt = None
         self.history = collections.deque((), 100)
+        self.mwlocked = False
```

Comment:
Oh, is this used at all?

Reply:
it's used by the "other" code (from MemoryView) which should probably go away as we recreate the menu in FuncGraphView